### PR TITLE
[FIX,DOCS] Prevent segfault when building docs when VTA is not available

### PR DESCRIPTION
**Pull request by tkonolige**
@hogepodge

### CODE REVIEW

**Comment by hogepodge** [comment]
Thanks! VTA has been a thorn for me for a while, so this helps. I'll also update my own environment to make sure it's installed.

**Review by tqchen — APPROVED** [review]

**Comment by tqchen** [comment]
This function seems to be not available in simular

File: vta/tutorials/frontend/deploy_classification.py [hunk 1]
```diff
@@ -63,6 +63,8 @@
 
 # Make sure that TVM was compiled with RPC=1
 assert tvm.runtime.enabled("rpc")
+# Make sure vta is available
+assert tvm.get_global_func("tvm.contrib.vta.init", True) is not None
```
Comment (tqchen) [inline vta/tutorials/frontend/deploy_classification.py:67]: This function is not available in simulator and only available in rpc server to initialize the remote bitstream

**Review by tqchen — CHANGES_REQUESTED** [review]

File: vta/tutorials/frontend/deploy_classification.py (hunk 1 above)
Comment (tkonolige) [reply vta/tutorials/frontend/deploy_classification.py:67]: What function can I check then?

Comment (tqchen) [reply vta/tutorials/frontend/deploy_classification.py:67]: I believe https://github.com/apache/tvm/pull/7761 already checks if vta binary is available

**Pull request closed without merging**